charts/jenkins/Chart.yaml

```diff
@@ -1,25 +1,26 @@
+---
 apiVersion: v2
 name: jenkins
 home: https://jenkins.io/
-version: 3.1.11
+version: 3.1.12
 appVersion: 2.263.3
 description: Jenkins - Build great things at any scale! The leading open source automation server, Jenkins provides hundreds of plugins to support building, deploying and automating any project.
 sources:
```

Comment:
yq wants to fix the indent issues, so i've done them manually here the first time